backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -245,6 +253,7 @@ public TestOrder editQueueItem(
 
     order.setDateTestedBackdate(dateTested);
 
+    _advisoryLockManager.unlock(AdvisoryLockManager.TEST_ORDER_LOCK_SCOPE, testOrderId.hashCode());
```

Comment:
What happens if an exception is thrown before this lock is released?

Reply:
Good question! Added try/finally blocks in a8f6012f